src/components/pages/calculator/Calculator.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/prefer-stateless-function */
+import { useState } from 'react';
+import calculate from '../../logic/calculate';
+/* eslint-disable */
+import ButtonsGrid from '../../buttons/Buttons';
+/* eslint-disable */
```

Comment:
- [ ] Please don't disable linter rules.

Reply:
@arslanbisharat Hi!, Thank you for the comments and the inline review, it's very helpful to understand it. 
I wanted if you can kindly suggest to me how I can fix this linter code. Because I was trying to fix it and I couldn't find the solution. It's in the line 
![image](https://user-images.githubusercontent.com/93630700/177634022-32553ff2-5424-4fa0-afc1-43f7f89ca909.png)

I tried to add the extension '.js' but then eslint told me to remove unexpected extensions, then when I remove them that errors come up. So I'm confused.